export const Failure = () => {
  console.log('A failure message!');
};
